components/server/src/ome/services/graphs/GraphOpts.java

```diff
@@ -96,7 +97,7 @@
      *
      * @param op Current {@link Op} to add to the stack
      * @param modified Whether or not the value was changed by the user
-     * @param details Active user login
+     * @param ec The event context.Active user login
```

Comment:
did you mean to keep the "Active user login"? If so, maybe adjust spacing/punctuation